src/Nox.Generator/Application/IntegrationEvents/DefaultIntegrationEvent.template.cs

```diff
@@ -12,4 +13,5 @@ namespace {{codeGeneratorState.ApplicationNameSpace}}.IntegrationEvents;
 /// <summary>
 /// {{entity.Name}}{{crudOperation}} integration event.
 /// </summary>
+[IntegrationEventType("{{crudOperation}}", nameof({{entity.Name}}))]
 internal record {{entity.Name}}{{crudOperation}}({{entity.Name}}Dto {{entity.Name}}) :  IIntegrationEvent;
```

Comment:
I know its now on your PR, but can you rename this to operation (we have done it in other templates)
```suggestion
internal record {{entity.Name}}{{operation}}({{entity.Name}}Dto {{entity.Name}}) :  IIntegrationEvent;
```